typescript/cli/src/commands/warp.ts

```diff
@@ -367,9 +372,21 @@ export const check: CommandModuleWithContext<{
       symbol,
     });
 
+    const warpCoreConfig = context.warpCoreConfig;
+
+    if (!warpCoreConfig) {
+      throw new Error('No warp core config found');
+    }
```

Comment:
seems like this should happen in the context function?

Reply:
IIUC, the type is just not expressive enough?